### PR TITLE
Eth verify mpt in all 3 languages

### PR DESCRIPTION
**Pull request by 0xRampey**

### CODE REVIEW

File: eth_verify/rust/native/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    // Deserialize the response to get block and transaction data
+    let block = serde_json::from_value(resp["result"].clone()).unwrap();
+    let result = calculate_mpt_root(block).await;
+    println!("Tx root generated and matched successfully: {}", result);
+    if !result {
+        println!("Tx root does not match");
+        return;
+    }
+
+    let header: Header = serde_json::from_value(resp["result"].clone()).unwrap();
```
Comment (aashutoshrathi) [inline eth_verify/rust/native/src/main.rs:45]: is the block same as header?

Comment (0xRampey) [reply eth_verify/rust/native/src/main.rs:45]: They have a lot of fields in common so can deserialise what we need to create Header

File: inputs/block.json
Comment (aashutoshrathi) [inline inputs/block.json:1]: let's use the same full data in block.json and let's not make any API calls

**Review by aashutoshrathi — APPROVED** [review]
- Let's add the missing piece from #28 in another PR

**Pull request merged**